domain-models-runtime/src/main/java/org/folio/rest/persist/PgUtil.java

```diff
@@ -430,6 +445,50 @@ public static void deleteById(String table, String id,
     }
   }
 
+
+/** Number of records to use from the title sort index in optimizedSql method */
```

Comment:
This comment

    /** Number of records to use from the title sort index in optimizedSql method */

does not have a variable it describes. Please remove this comment.